apps/happy-blocks/src/pricing-plans/components/billing-options.tsx

```diff
@@ -1,34 +1,76 @@
-import { createInterpolateElement } from '@wordpress/element';
+import { TERM_ANNUALLY, TERM_MONTHLY } from '@automattic/calypso-products';
+import { sprintf, __ } from '@wordpress/i18n';
 import { FunctionComponent } from 'react';
-import { PricingPlanBilling } from '../types';
+import { BlockPlan } from '../hooks/pricing-plans';
 
 const Promo: FunctionComponent = ( { children } ) => (
-	<span className="hb-pricing-plans-embed__promo">({ children })</span>
+	<span className="hb-pricing-plans-embed__promo">{ children }</span>
 );
 
 interface Props {
-	billings: PricingPlanBilling[];
+	plans: BlockPlan[];
 	value: string;
-	onChange: ( value: string ) => void;
+	onChange: ( value?: string ) => void;
 }
 
-const BillingOptions: FunctionComponent< Props > = ( { billings, value, onChange } ) => {
+const getAnnyallyDiscount = ( annuallyPlan?: BlockPlan, monthlyPlan?: BlockPlan ) => {
```

Comment:
Typo in "Annyally".

What do you think about calling it `annualPlan`, btw? "Annually" looks weird in this context. "monthly" is fine.